src/protect/test/unit/test_file_downloads.py

```diff
@@ -102,5 +103,49 @@ def _download_files(job):
                 raise
 
 
+    def test_file_downloads_from_https(self):
+        """
+        Test the functionality of get_file_from_url https
+        """
+        a = Job.wrapJobFn(self._download_https_files)
+        Job.Runner.startToil(a, self.options)
+
+
+    @staticmethod
+    def _download_https_files(job):
```

Comment:
This test is currently downloads the https s3 link using get_file from s3. Rename this test to be more informational and then write an additional test that tests an https link that is downloaded directly with urllib.urlopen2